.github/project_workflows/deploy_app_store.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         fetch-depth: 0
 
     - name: Run SwiftLint
-      uses: norio-nomura/action-swiftlint@3.1.0
+      uses: norio-nomura/action-swiftlint@master
```

Comment:
TBH, I'd prefer to provide an explicit tag (i.e., `@3.2.1`) rather than `master`.

It might have breaking changes in the future (although a GitHub action is less likely, it still is a software 😅 ). Plus, we may not have to update as frequently as we think; we can ignore the patch updates (`3.2.x`), for example.

Reply:
How about using this solution, it works great on Smashburger and we can fix the version we want.
```
- name: Run SwiftLint
      uses: docker://norionomura/swiftlint:0.53.0_swift-5.7
      with:
        args: swiftlint --strict
```

Reply:
No concern for me using Mark's approach.